src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const rows = table.rows;
+const head = rows[0];
+const people = [];
+const tBody = table.tBodies[0].children;
+
+for (const row of rows) {
+  const person = {};
+
+  person.name = row.cells[0].textContent;
+  person.position = row.cells[1].textContent;
+  person.age = row.cells[2].textContent;
+  person.salary = row.cells[3].textContent;
+  people.push(person);
+}
+people.pop();
+people.shift();
+
+function salaryToNum(str) {
+  const arr = str.split(',');
+  const value = arr[0].slice(1) + arr[1];
```

Comment:
The `salaryToNum` function assumes the salary string is formatted with a comma, e.g., '$1,000'. If the format changes or is inconsistent, this could lead to errors. Consider making this function more robust to handle different formats.